.github/labels.yml

```
@@ -0,0 +1,13 @@
+- name: "Package Issue"
+  color: "EB6420"
+  description: "Issues that are related directly to packages, and no changes to Chocolatey CLI is required."
+- name: "PowerShell Internal Host"
+  color: "D4C5F9"
+  description: "Issues that are related to how Chocolatey CLI spawns an internal PowerShell Host in order to execute the Chocolatey packaging scripts."
+- name: "Requires NuGet.Client Change"
```

To make it consistent with `shimgen`, and since it it a dependency as well, maybe it would be more appropriate to change this to?


```suggestion
- name: "Dependency - NuGet.Client"
```